mne/beamformer/_lcmv.py

```diff
@@ -39,27 +50,38 @@ def _reg_pinv(x, reg):
     return linalg.pinv(x), d
 
 
-def _setup_picks(picks, info, forward, noise_cov=None):
-    if picks is None:
-        picks = pick_types(info, meg=True, eeg=True, ref_meg=False,
-                           exclude='bads')
+def _setup_picks(info, forward, data_cov=None, noise_cov=None):
+    # get a list of all channel names:
+    fwd_ch_names = set([c['ch_name'] for c in forward['info']['chs']])
```

Comment:
while we're at it let's drop Pyhon 2.6 support, ```{c['ch_name'] for c in forward['info']['chs']}``` does the job. OR  ```set(c['ch_name'] for c in forward['info']['chs'])```, more explicitly.